DataFormats/BTauReco/src/TaggingVariable.cc

```diff
@@ -127,7 +127,7 @@ const char* const TaggingVariableTokens[] = {
   /* [trackPtRatio]                             = */ "trackPtRatio",
   /* [trackPParRatio]                           = */ "trackPParRatio",
 
-  /* [trackIp2dVal]                             = */ "trackIp2dVal",
+  /* [trackIp2dSig]                             = */ "trackIp2dSig",
```

Comment:
Removing old tagging variables could lead to backward incompatibilities. Please only add new variables. However, a bigger problem is that this change is inconsistent. Why would you just change the token but not the corresponding description. Also, where are the corresponding changes in the header file?